src/main/java/com/porpapps/wealthwatchapi/model/Portfolio.java

```diff
@@ -1,16 +1,58 @@
 package com.porpapps.wealthwatchapi.model;
 
+import jakarta.persistence.*;
 import lombok.Data;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
 
 import java.time.LocalDateTime;
 
 @Data
+@Entity
+@Table(name = "portfolio")
 public class Portfolio {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id",
+            columnDefinition = "INT",
+            nullable = false,
+            updatable = false)
     private Integer id;
+
+    @ManyToOne(targetEntity = User.class, cascade = CascadeType.ALL)
+    @JoinColumn(name = "user_id",
+            referencedColumnName = "id",
+            columnDefinition = "INT",
+            nullable = false,
+            updatable = false)
     private User user;
-    private String name;
-    private Enum currency;
+
+    @Column(name = "name",
+            columnDefinition = "VARCHAR(50)",
+            nullable = false,
+            unique = true)
+        private String name;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "currency", nullable = false)
```

Comment:
I think you missed the `columnDefinition` in this column

Reply:
done.